spacemesh/v1/types.proto

```diff
@@ -95,6 +95,7 @@ message Layer {
         LAYER_STATUS_UNSPECIFIED = 0; // not yet approved or confirmed
         LAYER_STATUS_APPROVED = 1; // approved by hare
         LAYER_STATUS_CONFIRMED = 2; // confirmed by tortoise
```

Comment:
you can probably remove CONFIRMED, it is useless now
i think it was working before the change where hare applied blocks immediately

Reply:
but i think it still has value in the future. for us, this is more like the layer is finalized.